src/PerformanceTests/PTODBCInfo/performance_odbc_info.cpp

```diff
@@ -83,10 +83,10 @@ void GetVersionInfoString(std::string& version_info) {
     ASSERT_TRUE(es_comm.ConnectDBStart());
 
     // Issue request
-    std::string endpoint, content_type, query, fetch_size;
+    std::string endpoint, content_type, query, fetch_size, cursor;
     std::shared_ptr< Aws::Http::HttpResponse > response = nullptr;
     es_comm.IssueRequest(endpoint, Aws::Http::HttpMethod::HTTP_GET,
```

Comment:
This is really awkward as mentioned - shouldn't IssueRequest return the response?